monkestation/code/modules/clothing/bodycams/Bodycamcode.dm

```diff
@@ -0,0 +1,145 @@
+
+/obj/item/clothing/suit/armor/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/bodycamera_holder)
+
+/obj/item/clothing/suit/hooded/wintercoat/security/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/bodycamera_holder)
```

Comment:
You could maybe move these to some other overwrite file, up to personal choice but it might look a little confusing to have this Initialize here